src/test/java/baseClasses/FBPageTest.java

```diff
@@ -23,6 +26,15 @@ public void demoTest()
 		driver.close();
 	}
 	
+	

```

Comment:
Please add @Test testng annotation and add this method to the testing.xml file as well

Reply:
Sure, I'll do that and revert back asap.

Reply:
Changes Done.